x-pack/plugins/security_solution/cypress/integration/value_lists.spec.ts

```diff
@@ -46,7 +46,6 @@ describe('value lists', () => {
     });
 
     afterEach(() => {
-      removeSignalsIndex();
```

Comment:
There's still both a `cleanKibana` and a `removeSignalsIndex` in this tests' setup.

Reply:
You are right! thanks for your check, I missed that one as well :D